tests/python_package_test/test_dask.py

```diff
@@ -273,7 +273,7 @@ def test_classifier(output, task, boosting_type, client):
     p2_proba = local_classifier.predict_proba(X)
     s2 = local_classifier.score(X, y)
 
-    if boosting_type == 'rf' and output == 'dataframe-with-categorical':
```

Comment:
I believe this part has already been removed in #4088. Please sync with the latest `master`.